a/html/oki/panel/index.html

```diff
@@ -8,7 +8,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link rel="stylesheet" href="assets/css/frontend.css"/>
-    <link rel="stylesheet" href="//cdn.rawgit.com/okfn/app-theme/924b03098a3d0fcaa5acf573f51571f519f80c94/css/main.min.css">
+    <link rel="stylesheet" href="//cdn.rawgit.com/okfn/app-theme/v2.0.0/css/main.css">
```

Comment:
@smth I'm merging but the fact that we are referencing the app theme like this leads me to believe that the app theme belongs here too.